### PR TITLE
Decant

### PR DESCRIPTION
**Pull request by vikke1234**

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java [hunk 1]
```diff
@@ -243,4 +243,40 @@ default boolean swapAssignment()
 	{
 		return true;
 	}
+<<<<<<< HEAD
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java]: You need to remove the merge conflict markers.

File: runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java [hunk 2]
```diff
@@ -243,4 +243,27 @@ default boolean swapAssignment()
 	{
 		return true;
 	}
+
+
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java]: Only one newline here.

File: runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java [hunk 3]
```diff
@@ -486,7 +485,20 @@ else if (target.contains("bob barter") && config.swapDecant())
 		}
 		else if (target.contains("zahur"))
 		{
-			swap(config.swapZahur().getOption().toLowerCase(), option, target, false);
+			switch (config.swapZahur())
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java]: To be honest, I preferred `swap(config.swapZahur().getOption().toLowerCase(), option, target, false);`. The performance hit is in a number of nanoseconds, and will not have any real impact on client functionality. I don't know how others feel about this, so I'll let others chime in if they want to.

**Comment by ollieatkinson** [comment]
Does this merge request conflict with the Decant menu entry swapper here: https://github.com/runelite/runelite/pull/4109?

**Comment by Adam-** [comment]
Yes, it does. This can be rebased specifically for the Zahur swapping.

**Comment by Nightfirecat** [comment]
This is a niche feature that is best suited for the plugin hub, and is currently covered by the Menu swapper extented plugin there. See runelite/plugin-hub#42

**Pull request closed without merging**